tasks/broad/Alignment.wdl

```diff
@@ -54,7 +54,7 @@ task SamToFastqAndBwaMemAndMba {
     set -o pipefail
     set -e
 
-    if [-z ${BWA_VERSION}]; then
+    if [ -z ${BWA_VERSION}]; then
```

Comment:
Might as well put a space on both sides 
```suggestion
    if [ -z ${BWA_VERSION} ]; then
```
I have also read that the double bracket syntax is supposed to be better. '[[' vs '['

Reply:
Thanks for catching that! `}]` is still invalid syntax, I made this PR in a hurry.

`[[` is a superset of `[`. I don't think it's necessary.